astroplan/plots/__init__.py

```diff
@@ -1,2 +1,10 @@
+"""
+`astroplan.plots` is a module for making plots of important quantities within
+the `astroplan` toolbox, such as airmass vs. time, etc.
+
+Note that, at minimum, you must construct `Observer`, `Target` and
+`astropy.time.Time` objects as input for all plots.
```

Comment:
@jberlanga - I would suggest not saying this part here, because it's not necessarily true in general - future plots might use other things as inputs.  I think this is pretty straightforward in the API docs (or it should be made so if it isn't!), so specific mentions don't really belong in the module level docstring. One option might be to re-word this to just point to the main `astroplan` package, saying something like "these plotting function use functionality and classes from the `astroplan` package, so familiarity with that is assumed when using these plotting functions."

However, perhaps something to add here instead would be references to the tutorials?  That is, something like ``for worked examples on how to use some of these functions, see the :doc:`name_of_tutorial` page.``?